src/linux/classes/com/alibaba/wisp/engine/WispEngine.java

```diff
@@ -529,7 +529,7 @@ void shutdown(WispControlGroup group) {
         public void run() {
             WispCarrier.current().runTaskInternal(
                     wispControlGroup == null ? new ShutdownEngine() : new ShutdownControlGroup(wispControlGroup),
-                    WispTask.SHUTDOWN_TASK_NAME, null, null);
+                    WispTask.SHUTDOWN_TASK_NAME, null, null, 0);
```

Comment:
Maybe we could just pass default STACK_SIZE here? 
0 is only used to represent default STACK_SIZE.

Reply:
Yes, we could. My thought is to not put `STACK_SIZE` everywhere and 0 obviously seems to be a default value.